js/components/scrubberBar.js

```diff
@@ -238,41 +239,56 @@ var ScrubberBar = React.createClass({
     }
 
     var thumbnailContainer = null;
+    var thumbnailCarousel = null;
+    var hoverTime = 0;
+    var hoverPosition = 0;
+    var hoveredIndicatorStyle = null;
+
     if (this.props.controller.state.thumbnails && (this.state.scrubbingPlayheadX || this.lastScrubX || this.state.hoveringX)) {
       if (this.state.scrubbingPlayheadX) {
-        var hoverPosition = this.state.scrubbingPlayheadX;
-        var hoverTime = (this.state.scrubbingPlayheadX / this.state.scrubberBarWidth) * this.props.duration;
+        hoverPosition = this.state.scrubbingPlayheadX;
+        hoverTime = (this.state.scrubbingPlayheadX / this.state.scrubberBarWidth) * this.props.duration;
         playheadClassName += " oo-playhead-scrubbing";
-      }
-      else if (this.lastScrubX) {//to show thumbnail when clicking on playhead
-        var hoverPosition = this.props.currentPlayhead * this.state.scrubberBarWidth / this.props.duration;
-        var hoverTime = this.props.currentPlayhead;
+
+        thumbnailCarousel =
+          (<div className="oo-scrubber-carousel-container">
```

Comment:
can we move `<div className="oo-scrubber-carousel-container">` into the `ThumbnailCarousel` component?